### DISCUSSION TITLE
Colab Reconnection Beginner

### DISCUSSION THREAD

**Discussion by Thyrome** [TheLastBen/fast-stable-diffusion#273] [Q&A]
Dear all,
 I've searched many places but could not find a good answer. If someone could point me in the right direction I would be very thankful. Forgive me as I am new to this.

I have installed the fast stable diffusion on a colab Notebook placed on my gdrive.

I get it to run beautifully and have generate my first pictures.  But then after having a Break I have problems restarting the previously working SD.
If I use the localhost and reopen the given  link at the end of the Installation process it give me a 404 error. If I do the same with Gato.app I get a 502 Bad gateway. 

Could anyone please explain me how to restart my working Notebook? Without the need to reinstall. 

I can see this being a beginner colab Problem.

Greatfull for any answer
And thankful for all the fine developers
Christian

**Comment by TheLastBen** [comment]
Hi,
if you disconnect from the notebook, after reconnecting, you'll need to rerun all the cells from the beginning, it'll take about 2-3 minutes.

**Reply by plasmasnake88** [reply]
Hi Ben,
does this mean that it's not possible to work with extensions?

**Reply by TheLastBen** [reply]
hi, most of the extensions work fine

**Reply by plasmasnake88** [reply]
hi ben, 
thanks for your hard work. happy to give you  coffee anytime
my question however is, how do i restart the session without having to run the entire script again and therefor overwrite the existing folders?

**Reply by TheLastBen** [reply]
Hi,
simply by entering the previous session name in the sessions cell, it will load the previous session with the instance images and the model.

**Reply by DigiClau** [reply]
Hi Ben

If you are still here, could you please explain where I can enter the previous session name?

**Reply by TheLastBen** [reply]
in the session_name input box in the "Creat/Load Session" cell
